lxqtscreensaver.cpp

```diff
@@ -168,6 +168,7 @@ void ScreenSaverPrivate::reportLockProcessError()
     }
     box->setText(message.arg(lock_command));
     box->exec();
+    delete box;
```

Comment:
How about using smart pointer or event stack allocated variable?

Reply:
Well the end result is the same.
Heap is not needed here, stack is fine.